### PR TITLE
[Doc] update kafka connector release notes

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/ecosystem_release/kafka_connector.md [hunk 1]
```diff
@@ -18,12 +18,40 @@ displayed_sidebar: "English"
...
+
+#### 1.0.1
+
+Release date: November 28, 2023
+
+**Features**
+
+- Support loading debezium data into Primary key tables. [#4](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/4)
+- Support parsing JSON data without schema registry. [#6](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/6)
+
```
Comment (amber-create) [inline docs/en/ecosystem_release/kafka_connector.md:54]: image
Added; Supports

File: docs/en/ecosystem_release/kafka_connector.md [hunk 2]
```diff
@@ -18,12 +18,40 @@ displayed_sidebar: "English"
...
+Support the source data to be of DECIMAL type. [#7](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/7)
+
+#### 1.0.1
+
+Release date: November 28, 2023
+
+**Features**
+
+- Support loading debezium data into Primary key tables. [#4](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/4)
```
Comment (amber-create) [inline docs/en/ecosystem_release/kafka_connector.md:52]: image
-> Primary Key

File: docs/zh/ecosystem_release/kafka_connector.md [hunk 3]
```diff
@@ -18,12 +18,40 @@ displayed_sidebar: "Chinese"
...
+
+添加开源协议为 Apache 协议。[#9](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/9)
+
+#### 1.0.2
+
+发布日期：2023 年 12 月 14 日
+
+**新增功能**
+
+支持源数据为 Decimal 类型。[#7](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/7)
```
Comment (amber-create) [inline docs/zh/ecosystem_release/kafka_connector.md:44]: Decimal -> DECIMAL

File: docs/en/ecosystem_release/kafka_connector.md (hunk 2 above)
Comment (evelynzhaojie) [inline docs/en/ecosystem_release/kafka_connector.md:52]: debezium -- Debezium

File: docs/en/ecosystem_release/kafka_connector.md [hunk 4]
```diff
@@ -18,12 +18,40 @@ displayed_sidebar: "English"
...
 
 ### 1.0
 
+#### 1.0.3
+
+Release date: December 19, 2023
+
+**Features**
+
+Add the open source license as Apache license. [#9](https://github.com/StarRocks/starrocks-connector-for-kafka/pull/9)
```
Comment (evelynzhaojie) [inline docs/en/ecosystem_release/kafka_connector.md:36]: Added Apache License for starrocks-connector-for-kafka.

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Review by zaorangyang — APPROVED** [review]

**Review by zaorangyang — APPROVED** [review]

**Pull request merged**